### ISSUE TITLE
Line info: Line numbers and selected line count

### ISSUE THREAD

**Issue by vertigo220** [streetwriters/notesnook#1815]
### What problem are you facing?

It's often useful to know how many lines a document has or how many lines a portion of it has.

### What's the solution you'd like to see in Notesnook?

I'd like optional line numbers on the left like some text editors have, e.g. Notepad++, as well as a count of total lines and how many lines are selected similar to how it currently shows word count.

**Comment by thecodrr** [comment]
Are you talking about the codeblock? Notesnook's editor is a rich text editor, not a code editor so there's no concept of "lines".

**Comment by vertigo220** [comment]
No, I'm talking in general. I'm not sure why there would be a difference. LibreOffice Writer and MS Word both know what lines are and can count them, and Word can even show line numbers. A line is an instance of a CR/LF.
